site/data/qasida/ya-kitab-al-ghuyub.json

```
@@ -53,6 +57,9 @@
     },
     "translation": {
         "en": [
+            ["The beloved's visage, the ultimate quest,"],
+            ["Who sees, knows, while others are suppressed."],
```

I'm not sure about the translation of the first half of this line... The meaning of the Arabic is 'Those who see know [...]', and I think that makes more sense in English than saying 'Who sees, knows [...]' 

Also, محجوب means 'veiled' (same root word as hijab), and I think that works better than 'suppressed' here too.. 

What do you think?

---

Those make sense to me!

I'm happy to prefer the more literal meanings for now, I think the AI is trying to get the rhyme in and that is probably why it chose those words tbh. 

At some point we should write up a bit of guideline for how we do translations to help us be consistent in them too

---

Yes, that would be really helpful! Let me know if you want to brainstorm the translation guidelines at all

---

```suggestion
            ["Those who see, know, while others are veiled."],
```

would that work then?



> Yes, that would be really helpful! Let me know if you want to brainstorm the translation guidelines at all

I can only be a facilitator to your knowledge but definitely yes

---

Yes, that works!

And also yes happy to brainstorm translation guidelines / ideas isA

(Side note: still need to figure out how to do those quote replies....)

---

I also got the following after tweaking the prompt to:

```
Using the perspective of a Cambridge University Academic who has studies Sufi Poetry poetically translate the snippet below, but prefer literal meaning over rhyme, returning it in the exact same form:
'''
            ["طَلْعَةُ الْمَحْبُوْبْ غَايَةُ الْمَطْلُوْبْ"],
            ["مَنْ رَأَى يَدْرِيْ وَالسِّوَى مَحْجُوْبْ"],
'''
```

```
"ara": [
    ["The appearance of the beloved is the ultimate sought-after goal,"],
    ["Whoever sees, knows, while others are veiled."]
],
```

---

Yes, this is a much better translation imo.

In the past, I've gone back and forth a bit with the AI - in the first prompt, I would ask for a translation that maintains the poetic form and rhythm, then I would ask it to tweak some parts to preserve the literal meaning, then I might ask it a third time to strike a balance somewhere in between. I've had different results with different platforms. I'll share those prompts isA and we can keep experimenting till we find a good flow....

---

Using the same prompt in Jasper.ai (which Sh Ma'n uses), this is the output:

> The countenance of the beloved is the pinnacle of desire,  
Whoever has seen knows, and all else remains veiled.

---

And here is a slight tweak to the prompt with ChatGPT:

```
Using the perspective of a Cambridge University Academic who has studied Sufi Poetry, translate the snippet below, maintaining the poetic effect, but prefer literal meaning over rhyme, returning it in the exact same form: ''' ["طَلْعَةُ الْمَحْبُوْبْ غَايَةُ الْمَطْلُوْبْ"], ["مَنْ رَأَى يَدْرِيْ وَالسِّوَى مَحْجُوْبْ"], '''
```


```
"Beholding the beloved’s visage is the ultimate desire,
Whoever sees knows, while others remain veiled."
```

---

Have also had good results with this prompt:

```
Following A. J. Arberry's style of translating Arabic poetry into English, translate the following snippet, returning it in the exact same form: 
```

---

Nice, they all sound fine at this point to me...

I tried the 'Write For Me' prompt in ChatGPT and got:

> ["The countenance of the Beloved is the ultimate desire,"]
> ["One who beholds understands, while all else remains veiled."]

Are we actually paying for Jasper AI ? Or is there some Uni free tier?

---

> Nice, they all sound fine at this point to me...


Ah, yeah, of course they all sound fine to me too - I just got excited about trying out different translation prompts, but probably should have saved those comments for the translation thread instead!

 
> Are we actually paying for Jasper AI ? Or is there some Uni free tier?

We're not paying for it - I wouldn't mind in principle, but since we seem to be getting similar results with ChatGPT, perhaps there's no need at this stage...

---

> (Flexing my quote replies..)

❤️ 

> We're not paying for it - I wouldn't mind in principle, but since we seem to be getting similar results with ChatGPT, perhaps there's no need at this stage...

Mhm, I was just wondering how Sh Ma'n is using it. As you say I think there isn't anything wildly different pushing up either way.


> Have also had good results with this prompt:

Somehow missed this one, this is very cool, we could get real spicy if we can get specific names

> for the translation thread instead!

Yeahh I think we can continue this in #24